src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+
+  if (days < 3) {
+    return days * cost;
+  }
+
+  if (days >= 3 && days < 7) {
+    return days * cost - 20;
+  }
+
+  if (days >= 7) {
+    return days * cost - 50;
+  }
```

Comment:
Agree with @jursylian , create variables for 20 and 50 values
Also don't repeat yourself: create one more variable for days * cost value